spec/factories/identity.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :identity do
+    user_id { user.id }
+    provider_uid { '134568678' }
+    provider_type { 'facebook' }
+    auth_hash { 'sample_token_hhdkfh2455' }
```

Comment:
Have you looked at a sample facebook auth hash? Is it just a string? I expected a response object with multiple values.